rinad/src/ipcp/ipc-process.cc

```diff
@@ -419,7 +414,7 @@ void IPCProcessImpl::processSetPolicySetParamRequestEvent(
                 result = security_manager_->set_policy_set_param(remainder,
                                                                 event.name,
                                                                 event.value);
-        } else if (component == "enrollment") {
+        } else if (component == "enrollment-task") {
```

Comment:
It's just that we had not taken the decision yet ("enrollment" or "enrollment-task"). Now we have and so we use "enrollment-task" uniformly.